### PR TITLE
Add sig-docs-bg team and members

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Adding Bulgarian localization teams and members

### CODE REVIEW

**Comment by OrlinVasilev** [comment]
@natalisucks yeah I though they did it already.. https://github.com/kubernetes/org/issues/4092 here is my issue :) thank tou

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -308,6 +308,22 @@ teams:
...
+    - atsonkov
+    privacy: closed
+  sig-docs-bg-reviews:
+    description: PR reviews for Bulgarian content
+    members:
+    - OrlinVasilev
+    - nickolaev
+    - iliyan-s-petkov
+    - atsonkov
+    privacy: closed
```
[the remark this answers is not in the document]
Comment (OrlinVasilev) [reply config/kubernetes/sig-docs/teams.yaml]: Hi @sftim thought so as well but, we have the pl then ru, then pr and pt... then zh and then vi.. so I thought we are not doing it like that anymore. Fixed added the Bulgarian team right after "sig-docs-blog-owners" as this team looks like not part of the localization teams! If needs to be above blog just drop a line! :) Thank you!

[the remark this answers is not in the document]
Comment (sftim) [reply config/kubernetes/sig-docs/teams.yaml]: Ah oops. Well, we _should_ collate in alphabetical order.

**Comment by natalisucks** [comment]
Given that Orlin is still working on becoming a member of the Kubernetes organization, I'll be closing this PR. It can be reoped when the requirements for becoming members in order to start a localization have been met. Thanks!

/close

**Pull request closed without merging**